src/app/signup/signup.component.ts

```diff
@@ -72,12 +72,12 @@ export class SignupComponent implements OnDestroy {
       this.profileApiService
         .postProfile({
           username: this.signupForm.value.username || '',
-          firstName: this.signupForm.value.firstName || '',
-          lastName: this.signupForm.value.lastName || '',
+          first_name: this.signupForm.value.firstName || '',
+          last_name: this.signupForm.value.lastName || '',
         })
         .pipe(
```

Comment:
Infinite loading loop after submitting signup page:
steps to reproduce

auth0 signup -> enter username etc. -> infinite loading

login flow works!